tests/resources/existing-resources/main.tf

```diff
@@ -0,0 +1,56 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source  = "terraform-ibm-modules/resource-group/ibm"
+  version = "1.1.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# Create Cloud Object Storage instance and a bucket
+##############################################################################
+
+module "cos" {
+  source                 = "terraform-ibm-modules/cos/ibm"
+  version                = "7.5.1"
+  resource_group_id      = module.resource_group.resource_group_id
+  region                 = var.region
+  cos_instance_name      = "${var.prefix}-cos"
+  cos_tags               = var.resource_tags
+  bucket_name            = "${var.prefix}-bucket"
+  retention_enabled      = false # disable retention for test environments - enable for stage/prod
+  kms_encryption_enabled = false
+}
+
+##############################################################################
+# Cloud Monitoring
+##############################################################################
+
+module "cloud_monitoring" {
+  source                  = "terraform-ibm-modules/observability-instances/ibm//modules/cloud_monitoring"
```

Comment:
extra slash
```suggestion
  source                  = "terraform-ibm-modules/observability-instances/ibm/modules/cloud_monitoring"
```

Reply:
@tyao117 The extra slash is the correct syntax required to call a submodule. See https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/pull/2